mail_tracking/models/mail_message.py

```diff
@@ -130,7 +135,12 @@ def _get_failed_message(self):
         for message in self:
             res.update({
                 message.id: message.mail_tracking_needs_action
-                and bool(message.mail_tracking_ids.filtered(
+                and (
+                    message.author_id.id == self.env.user.partner_id.id
+                    or any(message.partner_ids.filtered(
+                        lambda x: x.id == self.env.user.partner_id.id))
+                )
+                and any(message.mail_tracking_ids.filtered(
                     lambda x: x.state in self.get_failed_states()))
             })
```

Comment:
Please make this more readable. See https://github.com/OCA/social/pull/447#discussion_r339488173 as it is pretty similar.